### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by Demirrr** [Accenture/AmpliGraph#219]
I was wondering whether (1) it is possible to use pretrained models and (2) do you plan to include any other KGE approaches published after 2018 ?

Cheers

**Comment by sumitpai** [comment]
For now, we can load the pretrained embeddings using the [constant initializer](https://docs.ampligraph.org/en/1.3.2/generated/ampligraph.latent_features.Constant.html#ampligraph.latent_features.Constant) and load the pretrained embeddings. You can fine tune it by training using a low learning rate
